src/Message/ApplePayAuthorizeResponse.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+class ApplePayAuthorizeResponse extends Response
+{
+    /**
+     * Is the response successful?
+     * Throws an exception if there's no code.
+     *
+     * @return boolean
+     * @throws \Omnipay\Common\Exception\InvalidResponseException
+     */
+    public function isSuccessful()
+    {
+        $statusCode = $this->getStatusCode();
+        return ($statusCode >= 200 && $statusCode < 300) || $statusCode == 304;
+    }
+
+    /**
+     * Is the response successful?
+     * Throws an exception if there's no code.
```

Comment:
what do you mean if there's no code? in general, having these functions return null instead of throwing an exception is what omnipay generally does

Reply:
i don't see where the exception is being thrown tho

Reply:
Copy and paste mistake. Fixing this.